contracts/src/bridge/L1/TenBridge.sol

```diff
@@ -18,6 +18,7 @@ contract TenBridge is
     ITenBridgeAdmin,
     AccessControl
 {
+    event Withdrawal(address indexed receiver, address indexed asset, uint256 amount);
```

Comment:
How do you know when it is a native transfer? What is the ``asset``?

Reply:
address(0)